lms/djangoapps/teams/views.py

```diff
@@ -830,6 +837,7 @@ def get(self, request):
                     CourseKey.from_string(course_key_string)
```

Comment:
Is there a way to take advantage of the fact that valid_courses may have already been limited to the single course supplied by the course_id param (line 812)?

I don't think that a course_id of None should be acceptable if a specific course_id was specified.